homeassistant/components/mvglive/sensor.py

```diff
@@ -100,38 +100,38 @@ class MVGLiveSensor(SensorEntity):
 
     def __init__(
         self,
-        station,
+        hass: HomeAssistant,
+        station_name,
         destinations,
-        directions,
         lines,
         products,
         timeoffset,
         number,
         name,
-    ):
+    ) -> None:
         """Initialize the sensor."""
-        self._station = station
         self._name = name
+        self._station_name = station_name
         self.data = MVGLiveData(
-            station, destinations, directions, lines, products, timeoffset, number
+            hass, station_name, destinations, lines, products, timeoffset, number
         )
-        self._state = None
+        self._state = "-"
```

Comment:
Why is this changed?
Setting the state to `None` will make it appear as "unknown" in the frontend, why do you change it?
Is it to keep it consistant with the default in the update method?